cloud/pkg/common/messagelayer/util.go

```diff
@@ -42,6 +42,7 @@ const (
 	ResourceDevice               = "device"
 	ResourceTypeTwinEdgeUpdated  = "twin/edge_updated"
 	ResourceTypeMembershipDetail = "membership/detail"
+	ResourceDeviceStateUpdated   = "/state/update"
```

Comment:
Not consistent with the logic above:  '/state/update', 'membership/detail'. Should you delete the prefix char '/'?